shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -100,34 +112,52 @@ class BottomLine extends PureComponent<Props> {
       // We want to use these emojis to render the correct custom icon.
       switch (this.props.snippetDecoration) {
         case '\u{1F4A5}': // Explosion (Collision) emoji (💥)
-          snippetDecoration = <Icon type="iconfont-boom" fontSize={26} />
+          snippetDecoration = (
+            <Icon
+              type="iconfont-boom"
+              fontSize={isMobile ? 40 : 28}
+              style={platformStyles({
+                common: {
+                  color: this.props.isSelected ? globalColors.white : globalColors.black_20,
+                },
+                isMobile: {
+                  marginTop: -10,
+                },
+              })}
+            />
+          )
           exploded = true
           break
         case '\u{1F4A3}': // Bomb emoji (💣)
-          snippetDecoration = <Icon type="iconfont-bomb" fontSize={isMobile ? 16 : 12} />
+          snippetDecoration = (
+            <Icon
+              color={globalColors.black_75}
+              type="iconfont-bomb"
+              fontSize={isMobile ? 16 : 12}
+              style={{alignSelf: 'flex-start'}}
+            />
+          )
           break
         default:
           snippetDecoration = this.props.snippetDecoration
       }
-
       content = (
-        <Box2 direction="horizontal" style={styles.bottomLineBox}>
-          {!exploded && (
-            <Markdown preview={true} style={style}>
-              {this.props.snippet}
-            </Markdown>
-          )}
+        <Box2 direction="horizontal" gap="xtiny" style={styles.bottomLineBox}>
           {!!snippetDecoration && (
             <Box2 direction="vertical" centerChildren={true}>
               {snippetDecoration}
             </Box2>
           )}
+          {!exploded && (
```

Comment:
You could fix the Flow error by adding `!exploded && this.props.snippet &&(` or by changing the props at the top from:

```
snippet: ?string,
```

to:

```
snippet?: string,
```

Reply:
Do not ever do a this.props.string test. Always do !! Or you'll crash android